Sources/Networking/Responses/RevenueCatUI/PaywallComponentsData.swift

```diff
@@ -88,6 +88,10 @@ public struct PaywallComponentsData: Codable, Equatable, Sendable {
     public var componentsLocalizations: [PaywallComponent.LocaleID: PaywallComponent.LocalizationDictionary]
     public var defaultLocale: String
 
+    // The ID used for deeplinking
+    @_spi(Internal)
```

Comment:
Hmm in android, the whole class is "@_spi", though I guess changing that now is a breaking change, so this might be the best compromise.